static/shared/js/emoji.js

```diff
@@ -170,7 +171,15 @@ export function update_emojis(realm_emojis) {
         }
     }
     // Add the Zulip emoji to the realm emojis list
+
+    // The Zulip emoji is the only emoji that uses a string ("zulip")
+    // as its ID. All other emoji use numeric IDs. We should avoid
+    // creating a special case like this as it's quite liable to cause
+    // bugs. We might be interested in giving it the ID of 0 (if that's
+    // safe to do?).
```

Comment:
I think our plan for the Zulip emoji is to convert it into a RealmEmoji at some point, but it's a big refactor that's probably not worth it for what we might game.